src/cli.mjs

```diff
@@ -104,18 +104,7 @@ function buildYargs(argvInput) {
     .env("FAUNA")
     .config("config", configParser)
     .middleware([checkForUpdates, logArgv], true)
-    .middleware((argv) => {
-      if (!argv.url) {
-        if (argv.local) {
-          argv.url = 'http://localhost:8443';
-        } else {
-          argv.url = 'https://db.fauna.com';
-        }
-      }
-      if (!argv.secret && argv.local) {
-        argv.secret = "secret";
-      }
-    })
+    .middleware(applyLocalArg)
     .middleware([fixPaths, buildCredentials], false)
```

Comment:
add it to the array on line 108? makes order clearer if they're all array-based or all standalone calls. e.g.:
```
.middleware([applyLocalArgs, fixPaths, buildCredentials], false)
```

Reply:
sure thing.

Reply:
done!